pkg/build/pipelines/python/import.yaml

```diff
@@ -12,16 +12,61 @@ inputs:
   import:
     description: |
       The package to import.
-    required: true
+    required: false
+  imports:
+    description: |
+      List of packages to import, separated by spaces.
```

Comment:
from https://github.com/chainguard-dev/melange/issues/1299, we had

```yaml
inputs:
  imports: |
    from libfoo import bar
    from asdf import other thing
    import bark
```

Reply:
Which do you prefer? Both could work

Reply:
Done